webroot/js/vocabulary/add-sentences.ctrl.js

```diff
@@ -67,7 +66,7 @@
                 method: 'POST',
                 url: rootUrl + '/vocabulary/save_sentence/' + id,
                 headers: {
-                    'X-CSRF-Token': csrfHeader
+                    'X-CSRF-Token': get_csrf_token()
```

Comment:
I havn’t checked that on my own, but I think this actually shouldn’t be needed at all because Angular already sets the X-CSRF-Token header from the cookie automatically, as specified in webroot/js/responsive/app.module.js (xsrfHeaderName and xsrfCookieName).